scripts/archtype.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+ARCH=$(uname -m)
+
+if [[ "$ARCH" = "x86_64" ]]; then
+    echo "amd64"
+elif [[ "$ARCH" = "armv6l" ]]; then
```

Comment:
y not || here then

Reply:
I think this was originally copy/pasted from go-algorand. Now that you've pointed it out, maybe it can be removed entirely.